api/src/main/resources/workbench.yaml

```diff
@@ -2346,6 +2394,35 @@ definitions:
         type: integer
         description: The Value of the annotation if the AnnotationType is INTEGER
 
+  ParticipantCondition:
```

Comment:
If we need to define an object type with all columns for all OMOP tables we want to display in the UI somewhere, our API is going to get rather large... and we're going to have to make tweaks to it over time to keep up with the CDM.

An alternative is to just do like:

"SELECT * FROM <table>"

and return arrays of dictionaries with the results. That's what the materialize cohort endpoint will do if you don't specify a list of specific columns.

I could see going either way, just wanted to get your take on this... and to see whether you think researchers would prefer to get back strongly typed objects like this.

If we *are* going to have objects for all of this in our API, we may want to consider having a separate YAML file for them that gets merged in.

Reply:
Chatted offline. This is similar to above -- providing nicely typed objects is easier for the client developer sorta, but makes our API grow rather large. For now, sounds fine to try out this approach, but we should consider the alternative down the road.

As long as we do have separate endpoints / objects for each table, I do still like the idea of putting the CDM tables in their own YAML file (though if you want to hold off for now, while we're figuring out the approach, no objections).

Reply:
Yep, SGTM.